src/classes/i18n.js

```diff
@@ -183,13 +226,26 @@ export default class i18n {
             this.languages.push(default_lang);
         }
 
-        return new Promise(resolve => {
+        await new Promise(resolve => {
             async.each(
                 this.languages,
                 this.get_lang_files.bind(this),
                 this.load_files.bind(this, resolve)
             );
         });
+
+        try {
+            // save version to VERSION file
+            return await fs_promises.writeFile(
+                this.current_version_path,
+                service_version
+            );
+
+        } catch (error) {
+            // can't save to file :(
+
+            return Promise.resolve();
```

Comment:
Not really sure why we need this try/catch here, if we can't load translation properly, shouldn't we just throw error and stop immediately?

Reply:
[`29707c0` (#14)](https://github.com/anyTV/anytv-i18n/pull/14/commits/29707c066497dfca965d679f91f50a92439d91ba)